modules/iasRtdProvider.js

```diff
@@ -29,10 +31,18 @@ function stringifySlotSizes(sizes) {
   return result;
 }
 
-function stringifySlot(bidRequest) {
+function stringifySlot(bidRequest, adUnitPath) {
+  var sizes = [];
+  if ('mediaTypes' in bidRequest) {
+    for (const type in bidRequest.mediaTypes) {
+      if (bidRequest.mediaTypes.hasOwnProperty(type) && type == 'banner') {
+        sizes.push(bidRequest.mediaTypes[type].sizes)
+      }
+    }
+  }
```

Comment:
Would the built in utils function for getAdUnitSizes help here? https://github.com/prebid/Prebid.js/blob/b4cf8b11e57e4df6b12582de68690d7fc1a25e8f/src/utils.js#L148

Reply:
@ChrisHuie I updated with the function getAdUnitSizes you suggested.